src/__main__.py

```diff
@@ -0,0 +1,4 @@
+"""Main file for project."""
+
+if __name__ == '__main__':
+    pass  # do main stuff
```

Comment:
This comment doesn't matter much, since it will be replaced, but let's get in to good habits:

- Sentence case, proper capitalisation.
- Comments which say that something needs to be done conventionally start `TODO:` or `FIXME:`.
```suggestion
    pass  # TODO: Run the app.
```